lib/blockscore/base.rb

```diff
@@ -15,11 +17,11 @@ def initialize(options = {}, &block)
     def attributes
       return @attributes if @loaded
       force!
-      @attributes
+      attributes
     end
 
     def id
-      @attributes.fetch(:id, nil)
+      attributes.fetch(:id, nil)
```

Comment:
Or potentially more generalized:

```
foo.fetch(bar, baz) -> foo[bar]
```

This will catch cases where the explicitly provided default equals the implicit one.